packages/mdc-base/package.json

```diff
@@ -4,8 +4,14 @@
   "version": "0.41.0",
   "license": "MIT",
   "main": "dist/mdc.base.js",
+  "module": "index.js",
+  "sideEffects": false,
+  "types": "dist/mdc.base.d.ts",
   "repository": {
     "type": "git",
     "url": "https://github.com/material-components/material-components-web.git"
+  },
+  "dependencies": {
+    "tslib": "^1.9.3"
```

Comment:
Does greenkeeper update dependencies in `packages/*/`? Or will we need to remember to update them all manually?

Reply:
Not sure - will have to look into it. If greenkeeper does not keep track of it, is there something you recommend?

Reply:
No idea lol

Reply:
I wouldn't be too inclined to worry about it - we have typescript at top-level, right? Any time we update that we could look at whether this has any updates.

Reply:
Renovate has a great support for monorepos.

Reply:
Thanks for the suggestion @merlinnot! We'll look into that 😀